src/Service.GraphQLBuilder/Mutations/CreateMutationBuilder.cs

```diff
@@ -232,7 +362,7 @@ private static NameNode GenerateInputTypeName(string typeName)
         /// <param name="returnEntityName">Name of type to be returned by the mutation.</param>
         /// <param name="rolesAllowedForMutation">Collection of role names allowed for action, to be added to authorize directive.</param>
         /// <returns>A GraphQL field definition named <c>create*EntityName*</c> to be attached to the Mutations type in the GraphQL schema.</returns>
-        public static FieldDefinitionNode Build(
+        public static IEnumerable<FieldDefinitionNode> Build(
```

Comment:
@rohkhann
can you chime in here?

Reply:
Waiting for Rohan's take on this. We can go with creating two build functions, but since most of the code will be same between the two, I'm thinking if we can continue with the current state.

Reply:
Please ping him explicitly if you need an answer for this, lets not delay checking in this code if its ready.

Reply:
We directly call the MutationBuilder.Build function. I think this create mutation field is called within the call to that function. However, in the context of this change @ayush3797 why do we need to return an array of field definition notes?

Reply:
Depending on whether `multiple-create` is enabled, we will get 1/2 mutation nodes. To keep the returned data structure agnostic of this feature flag, I changed it to return a list of nodes. The list size will be either 1 or 2.